main.py

```diff
@@ -65,6 +70,7 @@ def change_read():
         new_read = ui.get_read_value()
         book.read = new_read
         book.save()
+        ui.message('You have read ' + book.title + ' by ' + book.author)
```

Comment:
Delete this line. I requested changes to this code in it's own pull request, so it'd be better to just not have it in this branch.